src/steamship/data/block.py

```diff
@@ -47,6 +47,9 @@ class Block(CamelModel):
     upload_type: Optional[
         BlockUploadType
     ] = None  # for returning Blocks as the result of a generate request
+    content: Optional[
```

Comment:
I originally did `content` for consistency with `Block.create()`, but I agree that making it different is better since there's such a high chance of confusion. I changed it.